site/topics/overview/overview.rst

```diff
@@ -333,87 +334,110 @@ Example
-    * :math:`n` cells in the array would be filled, representing the queen locations
-    * This would be a very *direct* representation
-    * It can represent all possible configurations
+* However, this would include many inadmissible solutions since each city should be visited once and only once
 
+    * Except for the starting city, which is visited twice since it is started and ended on
 
-* Alternatively, a 1D array could be used
 
-    * It would require an :math:`n` length array
-    * Each value in the array could be a tuple representing the :math:`(x, y)` coordinate of each queen
-    * This is still relatively direct, but does need a bit of translation to represent the phenotype
-    * This can also represent all possible configurations
+* This knowledge can be taken advantage of
+* Instead, a permutation of the integers from :math:`0` to :math:`(n-1)` could be used
+* This would ensure each city is visited once and only once
 
+* To analyze the size of the search space, consider the number of permutations there are of the :math:`n` cities
 
-* The representation can be whatever one wants, but being clever about the encoding can have an impact on performance
+    * The starting/ending city is fixed, meaning there are a total of :math:`n-1` possibilities for the next city
+    * After that, there are a total of :math:`n-2` possible cities to visit
+    * Then :math:`n-3`, then :math:`n-4`, and so on
+    * Thus, there a total of :math:`(n-1) \times (n-2) \times (n-3) \times ... \times 2 \times 1` permutations
+    * Which can be written as :math:`(n-1)!`
+
+
+* Further, half of those permutations are reverse of the other half, so really it's :math:`\frac{(n-1)!}{2}`
+
+* This is still a very large number, but it is an improvement over :math:`n^{n}`
+
+* Ultimately however, the representation can be whatever, but being clever about the encoding can impact the performance
 
 
 Population
 ----------
 
-* Assuming the use of the 2D array encoding, the population would be a collection of 2D arrays
-* Alternatively, with the 1D array, the population would be a collection of 1D arrays
 * Whatever encoding is used, the population would simply be a collection of coded values
+* With the permutation representation, the population would be a collection of permutations of length :math:`n`
 
 
 Fitness
 -------
 
-* As discussed previously, there is value in turning the :math:`n` queens problem into an optimization problem
-
-    * Instead of looking for a valid board configuration, minimize the number of conflicts
-    * This provides a gradient
+* The phenotype is the actual Hamiltonian cycle
+* The genotype is encoding a Hamiltonian cycle as a permutation
 
+* The fitness would be the total length of the Hamiltonian cycle
+* Given a chromosome, sum up the distances between the cities in order
 
-* An appropriate fitness function for this problem would be to count the number of conflicting queens
```

Comment:
I'd still be explicit about the distance being the gradient here. It is more obvious than n-queens, but there's no need to force students to read between the lines for an introductory example.